clang/include/clang/AST/DeclarationName.h

```diff
@@ -682,6 +682,11 @@ class DeclarationNameTable {
   DeclarationName getCXXLiteralOperatorName(const IdentifierInfo *II);
 };
 
+struct CXXOperatorSourceInfo {
+  SourceLocation::UIntTy BeginOpNameLoc;
+  SourceLocation::UIntTy EndOpNameLoc;
```

Comment:
Curious why this is ::UIntTy instead of just the source location?  I realize it was before, but now that we're indirecting the object, I wonder if the original reason still exists.